optaplanner-wb-webapp/pom.xml

```diff
@@ -1094,9 +1094,9 @@
             <compileSourcesArtifact>org.drools:drools-wb-workitems-editor-api</compileSourcesArtifact>
             <compileSourcesArtifact>org.drools:drools-wb-workitems-editor-client</compileSourcesArtifact>
 
-            <!-- KIE UberFire extensions -->
-            <compileSourcesArtifact>org.kie.uberfire:kie-uberfire-social-activities-api</compileSourcesArtifact>
-            <compileSourcesArtifact>org.kie.uberfire:kie-uberfire-social-activities-client</compileSourcesArtifact>
+            <!-- UberFire extensions -->
```

Comment:
@ederign `<!-- UberFire extensions -->` are already defined on l. 1101, please include the artifact there.